VirtualH89/Src/AddressBus.cpp

```diff
@@ -41,82 +33,36 @@ AddressBus::readByte(WORD addr,
         return ic_m->readDataBus();
     }
 
-    BYTE index = addr >> pageSizeBits_c;
-
-    debugss(ssAddressBus, ALL, "%s: addr(%d), index(%d)\n", __FUNCTION__, addr, index);
+    // Could do this entirely inside MemoryLayout, but this way we have
```

Comment:
Yes, seems reasonable to make a readByte(addr) and keep the other for debugging/core-dump purposes.